BinaryObjectScanner/Protection/StarForce.cs

```diff
@@ -52,6 +52,11 @@ public class StarForce : IExecutableCheck<PortableExecutable>, IPathCheck
                 return $"StarForce {pex.GetInternalVersion()}";
             else if (name.OptionalContains("Protection Technology")) // Protection Technology (StarForce)?
                 return $"StarForce {pex.GetInternalVersion()}";
+            
+            // FrontLine ProActive (digital activation), samples https://dbox.tools/titles/pc/46450FA4/ https://dbox.tools/titles/pc/4F430FA0/ https://dbox.tools/titles/pc/53450FA1/
```

Comment:
Please manually wrap samples onto new comment lines

Reply:
Should be fixed now.